### DISCUSSION TITLE
Automatic antivirus scan setup?

### DISCUSSION THREAD

**Discussion by awooooool** [subhra74/xdm#622] [Q&A]
Hi, I'm kinda confused with setting up automatic antivirus scan. Is there any documentation about it?

I'm using Kaspersky Total Security 2021. I need to use "avp.exe SCAN [filename]" for command line scan. How do I put it in XDM?

Thanks in advance

**Comment by tEctutoRIt** [comment]
I too had similar question but using Microsoft Defender. Though I guess this discussion is dead?

**Comment by artemgur** [comment]
It seems that I figured out how to make it work (at least on C# beta, version 8.0.29).
It simply appends the file path to command line parameters, like this:
`[Antivirus executable] [Command line parameters] [Downloaded file path]`
As an example, working configuration for Kaspersky:
![image](https://github.com/user-attachments/assets/3de89a89-1cc3-4e36-9d74-b50002649a1e)
In this example, it seems to execute the following command:
`C:\Program Files (x86)\Kaspersky Lab\Kaspersky 21.17\avp.com SCAN "C:\path\to\file"`
To check that antivirus scan is indeed working, you can temporarily enable logging. Example for Kaspersky:
![image](https://github.com/user-attachments/assets/a0d8370e-020d-4a11-8889-6c55830fce2b)
It will create log.txt file on desktop (obviously you will need to replace my username with your username) with information about the scan.

**Comment by anas274** [comment]
Exciting Affiliate Partnership Opportunity with InstallCPM
Dear,

I hope you're doing well! My name is Muhammad Anas Mukhtar, and I represent InstallCPM, a leading affiliate network. We are expanding our partnerships and would love to explore the possibility of working with you to boost your traffic monetization.

At InstallCPM, we offer highly competitive payouts, with rates of up to $3 per install. We support a range of advertising formats, including download button banners, PHP scripts, and more, ensuring seamless integration with your traffic sources.

Key Benefits:

Competitive Payouts: Up to $3 per install.
Flexible Payment Terms: Choose from weekly or biweekly payments.
Multiple Payment Methods: Payoneer, WebMoney, Bitcoin, and direct bank transfers.
Reliable Support: We’re here to help you optimize and scale your earnings.
If this sounds like something you’d be interested in, I’d love to chat further and answer any questions you may have.

 You can easily reach me through:
WhatsApp: +923068119424
Email: muhammadanshh@gmail.com
skype: muhammadanshh@gmail.com
Telegram: +923068119424
Discord: https://discord.gg/QmBN7qjQ
Wechat: +923068119424
Linkedin: http://linkedin.com/in/muhammad-anshh-846365240
Thank you for considering this opportunity. I look forward to hearing from you and exploring how we can collaborate for mutual success!

Best regards,
Muhammad Anas Mukhtar
